program/Project Descriptions and Plans/CV32E40Pv2/Project_Launch_for_CV32E40Pv2_Nov_22_2021.md

```diff
@@ -0,0 +1,198 @@
+# OpenHW Project Launch Proposal: CV32E40Pv2
+*The PL proposal explains the "what". Some of it can be updated directly from the PC proposal* 

```

Comment:
This line is information for this document's author - it can probably be removed now.

Reply:
Ok. I'll remove it.